Tests/Validator/EnumValidatorTest.php

```diff
@@ -84,13 +84,28 @@ public function testInvalidBasketballPositionType()
             'entity' => 'Fresh\DoctrineEnumBundle\Tests\Fixtures\DBAL\Types\BasketballPositionType',
         ]);
 
+        $constraintValidationBuilder = $this->getMockBuilder('Symfony\Component\Validator\Violation\ConstraintViolationBuilder')
+                                            ->disableOriginalConstructor()
+                                            ->getMock();
+
+        $constraintValidationBuilder
+            ->expects($this->once())
+            ->method('setParameter')
+            ->with(
+                $this->equalTo('{{ value }}'),
+                $this->equalTo('"Pitcher"')
+            )->will($this->returnSelf());
+
+        $constraintValidationBuilder
+            ->expects($this->once())
+            ->method('setCode')
+            ->will($this->returnSelf());
```

Comment:
ExecutionContext also works differently even if the validators themselves haven't changed, hence the extra mocks.